src/main/kotlin/com/ampnet/mailservice/service/impl/LinkResolverServiceImpl.kt

```diff
@@ -53,5 +53,8 @@ class LinkResolverServiceImpl(applicationProperties: ApplicationProperties) : Li
         return "$baseUrl/$coop/$typePath".removeDoubleSlashes()
     }
 
+    override fun getProjectFullyFundedLink(coop: String, organizationUUid: String, projectUuid: String) =
+        "$baseUrl/$coop/$manageWithdrawalsPath/$organizationUUid/$manageProjectPath/$projectUuid".removeDoubleSlashes()
```

Comment:
Are you sure this is the correct path? This implementation produces an incorrect link, check on staging: "https://staging.ampnet.io/ampnet/dash/manage_withdrawals/2850aab6-12fd-4b81-98b7-cb00efda2a09/manage_project/fb41fdb9-e0ad-43e8-a7ec-be9bf6a0503a"

Looks to me that the correct path would be: "https://staging.ampnet.io/ampnet/dash/manage_groups/2850aab6-12fd-4b81-98b7-cb00efda2a09/manage_project/fb41fdb9-e0ad-43e8-a7ec-be9bf6a0503a" ->
"$baseUrl/$coop/$organizationInvitesPath/$organizationUUid/$manageProjectPath/$projectUuid"

Rename `organizationInvitesPath` to `manageOrganizationPath`.